content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -477,6 +477,13 @@ In the CLI, the access modes are abbreviated to:
 * RWX - ReadWriteMany
 * RWOP - ReadWriteOncePod
 
+{{< note >}}
+The access modes are used to match PersistentVolume and PersistentVolumeClaim.
```

Comment:
```suggestion
Kubernetes uses volume access modes to match PersistentVolumeClaims and PersistentVolumes.
In some cases, the volume access modes also constrain where the PersistentVolume can be mounted.
Volume access modes do **not** enforce write protection once the storage has been mounted.
```

Reply:
@sftim 
Please let me confirm.
Is your suggestion to replace all of my Line 481-484, not just my Line 481?
Is my understanding right?

Reply:
This suggestion does match my proposed change - it would replace one line with 3.

Reply:
@sftim 
I commited your suggestion.
Could you confirm my update?